distribution/lib/Standard/Table/0.0.0-dev/src/Io/File_Format.enso

```diff
@@ -0,0 +1,47 @@
+from Standard.Base import all
+from Standard.Base.Error.Problem_Behavior as Problem_Behavior_Module import Problem_Behavior
+from Standard.Base.Data.Text.Encoding as Encoding_Module import Encoding
+
+## This type needs to be here allow for use of Standard.Table functions.
+   Ideally, it would be an interface within Standard.Base and expanded by
+   additional implementations in Standard.Table.
+
+## Determines the format of file to use based on the file extension.
+type Auto
+    type Auto
+
+    ## ADVANCED
+       Gets the underlying File_Format for the specified file
+    materialise : File->File_Format
+    materialise file =
```

Comment:
The name of this function seems weird to me. Any particular reason for picking that one?